Karumi/Shot
# Break the build

### :pushpin: References
* **Issue:** Fixes #6 and fixes #8
* **Related PRs:** Please, review #12 first.

### :tophat: What is the goal?

Improve the feedback shown to the user while recording screenshots or passing the tests.

### How is it being implemented?

We've updated our core component named ``Shot`` to show some improved feedback to the user. Now, if a comparison fails, you can see the absolute path to both files. In the case where the screenshot is not found or the screenshots dimensions are different, you can see the exact dimension of every file. Here you have some screenshots:

* Message shown to the user while pulling the screenshots:

![screen shot 2017-08-21 at 11 32 23](https://user-images.githubusercontent.com/4030704/29515114-5604ad9a-866b-11e7-9a19-7da9950151c4.png)

* Message shonw to the user after recording the screenshots:

![screen shot 2017-08-21 at 11 41 06](https://user-images.githubusercontent.com/4030704/29515133-6c666100-866b-11e7-8d4e-d105f7cc3923.png)

* Message shown to the user after passing the tetsts:

![screen shot 2017-08-21 at 11 36 39](https://user-images.githubusercontent.com/4030704/29515148-8508fae2-866b-11e7-83cc-9debddd6376c.png)

* Message shown to the user if there are some tests failing due to different reasons:

![screen shot 2017-08-21 at 11 33 34](https://user-images.githubusercontent.com/4030704/29515155-9101b62c-866b-11e7-9a03-1e653413ba23.png)

### How can it be tested?

Checkout this branch, comment some consumer code, delete an already recorded screenshot or resize it and verify your screenshots.
